src/math/Matrix4.js

```diff
@@ -833,7 +833,7 @@ Matrix4.prototype = {
 
 	}(),
 
-	makeFrustum: function ( left, right, bottom, top, near, far ) {
+	makePerspective: function ( left, right, bottom, top, near, far ) {
```

Comment:
I was unsure about that. On the other hand, this is the right moment to correct this inconsistency. I'll change the code.